### PR TITLE
Added valid prefixes to Release Notes

### PR DESCRIPTION
**Pull request by kgal-pan**
## Status
Ready

## Description
Improving the documentation of the release notes

### CODE REVIEW

File: docs/documentation/release-notes.md [hunk 1]
```diff
@@ -110,14 +110,32 @@ One should specify in the corresponding release notes file the following changes
     - Bad example: `Added the timeout parameter.`  
     - Good example: `Added the timeout parameter, which enables you to define the amount of time (in minutes) that the integration will try to execute commands before it throws an error.`
 
-- If this is a single line release note, there is no need for the bullet point, just a regular sentence. 
+- If this is a single line release note, there is no need for the bullet point, just a regular sentence.
```
Comment (dorschw) [inline docs/documentation/release-notes.md:113]:
```suggestion
- Single line release notes do not need a bullet point.
```

File: docs/documentation/release-notes.md [hunk 2]
```diff
@@ -110,14 +110,32 @@ One should specify in the corresponding release notes file the following changes
...
+- The release note line must start with one of the following prefixes:
```
Comment (dorschw) [inline docs/documentation/release-notes.md:114]:
```suggestion
- Release notes must start with one of the following prefixes:
```

File: docs/documentation/release-notes.md [hunk 3]
```diff
@@ -110,14 +110,32 @@ One should specify in the corresponding release notes file the following changes
...
+
+```
+'Added support for...'
+'Added the...'
+'Fixed an issue...'
+'Improved implementation...'
+'Updated the Docker image to...'
+'You can now...'
+'Deprecated. '
+'Deprecated the...'
```
Comment (dorschw) [inline docs/documentation/release-notes.md:124]:
```suggestion
'Added support for '
'Added the '
'Fixed an issue '
'Improved implementation '
'Updated the Docker image to '
'You can now '
'Deprecated. '
'Deprecated the' 
```

as the `...` are not part of the prefix

File: docs/documentation/release-notes.md [hunk 4]
```diff
@@ -110,14 +110,32 @@ One should specify in the corresponding release notes file the following changes
...
+```
+
+Specifying any other prefix will result in an error when running `demisto-sdk doc-review`:
```
Comment (dorschw) [inline docs/documentation/release-notes.md:127]:
```suggestion
Release notes not using the prefixes will result in an error when running `demisto-sdk doc-review`:
```

**Comment by tschanfeld** [comment]
Edited file for language and formatting

**Comment by kgal-pan** [comment]
@dorschw - Can you force merge? The build failure is caused by https://github.com/demisto/etc/issues/47753.

**Comment by kgal-pan** [comment]
@jrosenpanw @tschanfeld - Please review and approve.

**Comment by tschanfeld** [comment]
@kgal-pan I added docs-approved label a few days ago, were there any changes after?

**Comment by kgal-pan** [comment]
> @kgal-pan I added docs-approved label a few days ago, were there any changes after?

Nope :)

File: docs/documentation/release-notes.md [hunk 5]
```diff
@@ -60,16 +58,16 @@ For single line RNs, follow this format:
...
 For single line RNs with a nested list, follow this format:
 ```markdown
 #### Integrations
 ##### Cortex XDR - IR
-  - Release note here.
-    - List item 1
-    - List item 2
+Release note here.
+  - List item 1
+  - List item 2
```
Comment (dorschw) [inline docs/documentation/release-notes.md:70]: @tschanfeld please revert this change to be a nested list (`- Release note here.`)

File: docs/documentation/release-notes.md [hunk 6]
```diff
@@ -60,16 +58,16 @@ For single line RNs, follow this format:
 ```markdown
 #### Integrations
 ##### Cortex XDR - IR
-  - Release note here.
+Release note here.
```
Comment (dorschw) [inline docs/documentation/release-notes.md:61]: @tschanfeld please revert this change to be a list (`- Release note here.`), as it matters for automated tests this document describes.

**Review by dorschw — CHANGES_REQUESTED** [review]

File: docs/documentation/release-notes.md (hunk 5 above)
Comment (dorschw) [reply docs/documentation/release-notes.md:70]: @kgal-pan please check which of the following works with `validate`:
```
- a
  - b
  - c
```
or
```
a 
  -  b
  - c
```

File: docs/documentation/release-notes.md (hunk 6 above)
Comment (dorschw) [reply docs/documentation/release-notes.md:61]: @kgal-pan please check whether this passes validations. If it does not, let's change it back.

**Comment by dorschw** [comment]
@kgal-pan fyi

File: docs/documentation/release-notes.md (hunk 5 above)
Comment (kgal-pan) [reply docs/documentation/release-notes.md:70]: @dorschw - Looks like both syntaxes pass validation:

```bash

> file=Packs/SecurityScorecard/ReleaseNotes/1_0_4.md
```

```bash
# Hyphen in root
> cat $file&&demisto-sdk validate -i $file --silence-init-prints

#### Integrations
##### SecurityScorecard
- Release note here
        - Fixed an issue with the *get_last_run* unit test.
        - Updated the Docker image to: *demisto/python3:3.10.4.28442*.
You are using demisto-sdk 1.6.4.

================= Validating file /Users/kgal/dev/demisto/content/Packs/SecurityScorecard/ReleaseNotes/1_0_4.md =================

Validating /Users/kgal/dev/demisto/content/Packs/SecurityScorecard/ReleaseNotes/1_0_4.md as releasenotes

The files are valid
```
```bash
# No hyphen in root
> cat $file&&demisto-sdk validate -i $file --silence-init-prints

#### Integrations
##### SecurityScorecard
Release note here
        - Fixed an issue with the *get_last_run* unit test.
        - Updated the Docker image to: *demisto/python3:3.10.4.28442*.
You are using demisto-sdk 1.6.4.

================= Validating file /Users/kgal/dev/demisto/content/Packs/SecurityScorecard/ReleaseNotes/1_0_4.md =================

Validating /Users/kgal/dev/demisto/content/Packs/SecurityScorecard/ReleaseNotes/1_0_4.md as releasenotes

The files are valid
```

**Review by dorschw — APPROVED** [review]
